tests/Rector/CodeQuality/ForeachToInArrayRector/Wrong/wrong.php.inc

```diff
@@ -0,0 +1,15 @@
+<?php declare(strict_types=1);
+
+final class MyClass
+{
+    public function foreachToInArray(): bool
+    {
+        foreach ($items as $item) {
+            if ($item === 'something') {
+                return true;
+            }
+        }
+
+        return false;
```

Comment:
@TomasVotruba I can't get rid of this `return false;` after the `foreach` loop. Can you help me?

```
carusogabriel@caruso:~/Code/rector$ vendor/bin/phpunit tests/Rector/CodeQuality/ForeachToInArrayRector/ForeachToInArrayRectorTest.php
PHPUnit 7.4.0 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.2.10-0ubuntu0.18.04.1
Configuration: /home/carusogabriel/Code/rector/phpunit.xml

F                                                                   1 / 1 (100%)

Time: 84 ms, Memory: 16.00MB

There was 1 failure:

1) Rector\Tests\Rector\CodeQuality\ForeachToInArrayRector\ForeachToInArrayRectorTest::test with data set #0 ('/home/carusogabriel/Code/rect...hp.inc', '/home/carusogabriel/Code/rect...hp.inc')
Original file "/home/carusogabriel/Code/rector/tests/Rector/CodeQuality/ForeachToInArrayRector/Wrong/wrong.php.inc" did not match the result.
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
     public function foreachToInArray(): bool\n
     {\n
         return in_array('something', $items, true);\n
+\n
+        return false;\n
     }\n
 }\n
 '

/home/carusogabriel/Code/rector/src/Testing/PHPUnit/AbstractRectorTestCase.php:84
/home/carusogabriel/Code/rector/tests/Rector/CodeQuality/ForeachToInArrayRector/ForeachToInArrayRectorTest.php:18
```

Reply:
@TomasVotruba Is the first example in the Rector class `src/Rector/CodeQuality/ForeachToInArrayRector.php` and the second one in the `Rector\Rector\AbstractRector`?

Reply:
Both in `ForeachToInArrayRector`.

Reply:
I got to laptop and updated the code snippet above, to make that clearer.

Reply:
@tomasvotruba No need for a rush. I'll finish it today